pkgs/development/python-modules/pyhocon/default.nix

```diff
@@ -1,38 +1,38 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-# Runtime inputs:
-, pyparsing
-# Check inputs:
-, pytest
 , mock
+, pyparsing
+, pytestCheckHook
+, python-dateutil
 }:
 
 buildPythonPackage rec {
   pname = "pyhocon";
-  version = "0.3.53";
+  version = "0.3.58";
 
   src = fetchFromGitHub {
     owner = "chimpler";
     repo = "pyhocon";
     rev = version;
-    sha256 = "1lr56piiasnq1aiwli8ldw2wc3xjfck8az991mr5rdbqqsrh9vkv";
+    sha256 = "sha256-ddspVDKy9++cISWH6R95r+gJrzNGqMTybI04OgVtIUU=";
   };
 
-  propagatedBuildInputs = [ pyparsing ];
+  propagatedBuildInputs = [
+    pyparsing
+    python-dateutil
+  ];
 
-  checkInputs = [ pytest mock ];
+  checkInputs = [
+    mock
+    pytestCheckHook
+  ];
 
-  # Tests fail because necessary data files aren't packaged for PyPi yet.
-  # See https://github.com/chimpler/pyhocon/pull/203
-  doCheck = false;
+  pythonImportsCheck = [ "pyhocon" ];
 
   meta = with lib; {
     homepage = "https://github.com/chimpler/pyhocon/";
     description = "HOCON parser for Python";
-    # Long description copied from
-    # https://github.com/chimpler/pyhocon/blob/55a9ea3ebeeac5764bdebebfbeacbf099f64db26/setup.py
-    # (the tip of master as of 2019-03-24).
```

Comment:
Please keep this attribution - it's the least we can do. :)

Reply:
I don't get why we should keep the comment. It seems to be the same source as for the other details, e.g. the same URL where the source is coming. 

I haven't seen any other derivation that is doing this.

Reply:
Well, I can tell you why I put it there in the first place. The long description is a verbatim quote, and as a diligent author, I cited it. Doing so reflects well on me and it's useful information anyway. If other packages are using verbatim quotes, perhaps they should cite them as well. 

I'd like to turn the question around, though. How does removing this citation help you achieve a version bump?

Reply:
That is literally the description on https://pypi.org/project/pyhocon/

We normally don't quote those things. The description might be adapted over time or changed to follow NixOS standards. Having a 3 line comment about this is unnecessary and just noise.